### DISCUSSION TITLE
code-server cli support --without-connection-token ?

### DISCUSSION THREAD

**Discussion by alanhe421** [coder/code-server#6128] [Q&A] [answered]
code-server --help can not see the arg.

I just see on some online-demo. and this https://github.com/microsoft/vscode/commit/2387375634512422255f1c5c2cbde2a886713188

**Comment by code-asher** [comment, marked as answer]
code-server does not allow setting 
--without-connection-token.  We always defaults to true since we 
have our own authentication mechanism.

https://github.com/coder/code-server/blob/c995988b70e4134dacfa98a8ae0af2b05c6cd7b6/src/node/routes/vscode.ts#L160

**Reply by code-asher** [reply]
`--auth none` achieves a similar result.
